SUMMARY.md

```diff
@@ -1,120 +1,121 @@
-      * [model.getInfo()](model/getInfo.md)
-      * [model.getInput(input, line)](model/getInput.md)
-      * [model.getInputsCount(input)](model/getInputsCount.md)
-      * [model.getLogicalSwitch(switch)](model/getLogicalSwitch.md)
-      * [model.getMix(channel, line)](model/getMix.md)
-      * [model.getMixesCount(channel)](model/getMixesCount.md)
-      * [model.getModule(index)](model/getModule.md)
-      * [model.getOutput(index)](model/getOutput.md)
-      * [model.getTimer(timer)](model/getTimer.md)
-      * [model.insertInput(input, line, value)](model/insertInput.md)
-      * [model.insertMix(channel, line, value)](model/insertMix.md)
-      * [model.resetTimer(timer)](model/resetTimer.md)
-      * [model.setCustomFunction(function, value)](model/setCustomFunction.md)
-      * [model.setGlobalVariable(index, flight_mode, value)](model/setGlobalVariable.md)
-      * [model.setInfo(value)](model/setInfo.md)
-      * [model.setLogicalSwitch(switch, value)](model/setLogicalSwitch.md)
-      * [model.setModule(index, value)](model/setModule.md)
-      * [model.setOutput(index, value)](model/setOutput.md)
-      * [model.setTimer(timer, value)](model/setTimer.md) [//]: <> (LUADOC-END:model)
-   * [Lcd Functions](lcd/lcd_functions.md) [//]: <> (LUADOC-BEGIN:lcd)
-      * [Lcd Functions Overview](lcd/lcd_functions-overview.md)
-      * [lcd.RGB(r, g, b)](lcd/RGB.md)
-      * [lcd.clear()](lcd/clear.md)
-      * [lcd.drawBitmap(bitmap, x, y [, scale])](lcd/drawBitmap.md)
-      * [lcd.drawChannel(x, y, source, flags)](lcd/drawChannel.md)
-      * [lcd.drawCombobox(x, y, w, list, idx [, flags])](lcd/drawCombobox.md)
-      * [lcd.drawFilledRectangle(x, y, w, h [, flags])](lcd/drawFilledRectangle.md)
-      * [lcd.drawGauge(x, y, w, h, fill, maxfill [, flags])](lcd/drawGauge.md)
-      * [lcd.drawLine(x1, y1, x2, y2, pattern, flags)](lcd/drawLine.md)
-      * [lcd.drawNumber(x, y, value [, flags])](lcd/drawNumber.md)
-      * [lcd.drawPixmap(x, y, name)](lcd/drawPixmap.md)
-      * [lcd.drawPoint(x, y)](lcd/drawPoint.md)
-      * [lcd.drawRectangle(x, y, w, h [, flags [, t]])](lcd/drawRectangle.md)
-      * [lcd.drawScreenTitle(title, page, pages)](lcd/drawScreenTitle.md)
-      * [lcd.drawSource(x, y, source [, flags])](lcd/drawSource.md)
-      * [lcd.drawSwitch(x, y, switch, flags)](lcd/drawSwitch.md)
-      * [lcd.drawText(x, y, text [, flags])](lcd/drawText.md)
-      * [lcd.drawTimer(x, y, value [, flags])](lcd/drawTimer.md)
-      * [lcd.getLastPos()](lcd/getLastPos.md)
-      * [lcd.refresh()](lcd/refresh.md)
-      * [lcd.setColor(area, color)](lcd/setColor.md) [//]: <> (LUADOC-END:lcd)
-   * [Bitmap Functions](Bitmap/Bitmap_functions.md) [//]: <> (LUADOC-BEGIN:Bitmap)
-      * [Bitmap.getSize(name)](Bitmap/getSize.md)
-      * [Bitmap.open(name)](Bitmap/open.md) [//]: <> (LUADOC-END:Bitmap)
+    * [Constants](constants.md)
+        * [Key Event Constants](key_events.md)
+    * [General Functions](general/general_functions.md)
```

Comment:
Careful here! The parts of this file are generated by a special script that parses OpenTX source files and generates documentation. 

This script needs `[//]: <> (LUADOC-BEGIN:general)` - this is a special marker for  it. It won't be visible in final book, but script searches for it.